opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -930,7 +931,7 @@ export const userAddRelation = async (context, user, userId, input) => {
   }
   // Check in case organization admins adds non-grantable goup a user
   const myGrantableGroups = R.uniq(user.administrated_organizations.map((orga) => orga.grantable_groups).flat());
-  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
+  if (!isAdminWithoutSettingsSetAccessCapability(user)) {
```

Comment:
should be `if (isAdminWithoutSettingsSetAccessCapability(user))`, without the `!`